Source/Core/DolphinWX/MainMenuBar.cpp

```diff
@@ -140,6 +140,7 @@ wxMenu* MainMenuBar::CreateMovieMenu() const
 
   movie_menu->Append(IDM_RECORD, _("Start Re&cording Input"));
   movie_menu->Append(IDM_PLAY_RECORD, _("P&lay Input Recording..."));
+  movie_menu->Append(IDM_STOP_RECORD, _("Stop Play/Recording Input"));
```

Comment:
Renamed it to Playing, but I'll see what I can do about renaming it depending on the action.